scripts/tests/cases/dc20_v2_e2e.py

```
@@ -0,0 +1,234 @@
+import os
+import sys
+import time
+import json
+import logging
+from datetime import datetime
+
+from nv_ingest_client.client import Ingestor
+from nv_ingest_client.util.milvus import nvingest_retrieval
+from nv_ingest_client.util.document_analysis import analyze_document_chunks
+
+# Import from interact module (now properly structured)
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+from interact import embed_info, milvus_chunks, segment_results, kv_event_log, pdf_page_count  # noqa: E402
+
+# Future: Will integrate with modular ingest_documents.py when VDB upload is separated
+
+# Suppress LazyLoadedList file not found errors to reduce terminal bloat
+lazy_logger = logging.getLogger("nv_ingest_client.client.interface")
+lazy_logger.setLevel(logging.CRITICAL)
+
+try:
+    from pymilvus import MilvusClient
+except Exception:
+    MilvusClient = None  # Optional; stats logging will be skipped if unavailable
+
+
+def _now_timestr() -> str:
+    return datetime.now().strftime("%Y%m%d_%H%M")
+
+
+def _get_env(name: str, default: str | None = None) -> str | None:
+    val = os.environ.get(name)
+    if val is None or val == "":
+        return default
+    return val
+
+
+def _default_collection_name() -> str:
+    # Make collection name configurable via TEST_NAME env var, default to dc20_v2
+    test_name = _get_env("TEST_NAME", "dc20_v2")
+    return f"{test_name}_{_now_timestr()}"
+
+
+def main() -> int:
```

This test case does not have any asserts. Is there a situation where this could fail silently but report as a success?

---

@jperez999 dc20_v2_e2e is our e2e integration test

---

right it is an integration test but are we checking for the results to ensure we are actually getting back what is exected? Do the other integration tests follow this same strategy? Seems here like you are returning 0 regardless of whether the results are the same or not. Shouldnt this fail if the results are different? IMO if you changed the if statement at the end to an assert you would be better off because it would fail if that assert failed.